package/yast2-add-on.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Jul 26 10:22:03 UTC 2018 - knut.anderssen@suse.com
+
+- Do not redraw when skipping the addons table (bsc#1102705)
```

Comment:
NP: may be "Do not show the main dialog when it is immediately skipped" is a bit better...